pkg/service/schema.go

```diff
@@ -214,7 +214,9 @@ func (ps *planSchemas) addParameterSchemas(
 			sips.UpdatingParametersSchema = ups
 		}
 		err := ups.Parameters.addProperties(instanceUpdateParameters)
-		log.Errorf("error building instance update param schema %s", err)
+		if err != nil {
```

Comment:
i rebased off my other open PR and this came with it, so review this in the other one.